Update aws-cloudtrail.md (#1671)
SumoLogic/sumologic-documentation: docs/cse/ingestion/aws-cloudtrail.md

### Configure an AWS CloudTrail Source

> 1. To configure a CloudTrail Source, see [Configure an AWS CloudTrail source](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudtrail-source/#configure-an-aws-cloudtrail-source).

Reviewer: ```suggestion
1. To configure a CloudTrail Source, see [Configure an AWS CloudTrail source](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudtrail-source/).
```

> 1. **Please configure Fields as shown below to forward cloudtrail logs to the CSE platform**.

Reviewer: ```suggestion
1. **Configure Fields as shown below to forward CloudTrail logs to the Cloud SIEM platform**.
```

> 1. To configure a CloudTrail Source, see [Configure an AWS CloudTrail source](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudtrail-source).

Reviewer 2: ```suggestion
1. To configure a CloudTrail Source, see [Configure an AWS CloudTrail source](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudtrail-source.md).
```

> 1. To configure a CloudTrail Source, see [Configure an AWS CloudTrail source](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudtrail-source.md).

Reviewer 2: ```suggestion
1. To configure a CloudTrail Source, see [Configure an AWS CloudTrail source](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source.md).
```

> 1. **Configure Fields** as shown below to forward CloudTrail logs to the Cloud SIEM platform**.

Reviewer 2: ```suggestion
1. Configure **Fields** as shown below to forward CloudTrail logs to the Cloud SIEM platform.
```

>     1. If you are not forwarding all sources in the hosted collector to CSE, click the **+Add Field** link, and add a field whose name is `_siemForward` and value is *true*. This will ensure all logs for this source are forwarded to CSE.

Reviewer: ```suggestion
    1. If you are not forwarding all sources in the hosted collector to CSE, click the **+Add Field** link, and add a field whose name is `_siemForward` and value is *true*. This will ensure all logs for this source are forwarded to Cloud SIEM.
```
